packages/init/src/steps/choose-or-create-a-repository.ts

```diff
@@ -118,13 +123,17 @@ export async function chooseOrCreateARepository(
   cwd: string,
   framework: Models.Frameworks,
   cookies: string,
-  base = DEFAULT_BASE
+  base = DEFAULT_BASE,
+  repoName?: string
 ): Promise<string> {
   const token = parsePrismicAuthToken(cookies);
   const repos = await Communication.listRepositories(token);
 
+  const hasRepo = repoName && repos.find((d) => d.domain === repoName);
+  if (hasRepo) return repoName;
```

Comment:
this fonction return repository domain and not repository name, so either the variable is not well named either the data is not the good one